ERCS/erc-2233.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2233
```

Comment:
```suggestion
eip: to be assigned
```
ERC numbers are assigned by ERC editors and associates, you don't get to self assign.  The minimum requirement for this is to have a specification.

Once a specification has been added a number can be assigned.